dax/errors.go

```diff
@@ -12,6 +12,7 @@ const (
 	ErrDatabaseIDExists         errors.Code = "DatabaseIDExists"
 	ErrDatabaseIDDoesNotExist   errors.Code = "DatabaseIDDoesNotExist"
 	ErrDatabaseNameDoesNotExist errors.Code = "DatabaseNameDoesNotExist"
+	ErrDatabaseNameExists       errors.Code = "DatabaseNameExists"
```

Comment:
Is this one used?
Maybe we should move both of the new error types here to the `dax` package (instead of `dax/schemar`).